src/app/zap-templates/zcl/data-model/chip/binding-cluster.xml

```diff
@@ -16,27 +16,22 @@ limitations under the License.
 -->
 <configurator>
   <domain name="CHIP" />
+
+  <struct name="BindingEntry">
+    <cluster code="0x001e"/>
+    <item name="nodeId" type="NODE_ID"/>
+    <item name="groupId" type="GROUP_ID"/>
+    <item name="endpointId" type="ENDPOINT_NO" />
+    <item name="clusterId" type="CLUSTER_ID" />
+  </struct>
```

Comment:
I don't know where this came from, but this does not match the spec.  The struct has a different name, it's supposed to have a fabric index, it's supposed to have optional fields, etc, etc.

Had this matched the spec, there would have been no need to add that fabric index argument at all, because it would be right in the struct, as required by the spec.